tests/unit/lexicon/test_config.py

```
@@ -0,0 +1,49 @@
+import pytest
+
+from vekna.lexicon import RitualDefinitionError
+from vekna.lexicon._links.loader import read_config
+from vekna.lexicon._pacts import Config, RitualsConfig
+
+
+class TestReadConfig:
+    @staticmethod
+    def test_reads_modules_and_files(tmp_path):
```

Ruff (0.15.21)

[warning] Missing return type annotation for staticmethod `test_reads_modules_and_files`

Add return type annotation: `None`

(ANN205)